Assets/Scripts/Player/PlayerGroundCheck.cs

```diff
@@ -0,0 +1,21 @@
+﻿using UnityEngine;
+using System.Collections;
+
+public class PlayerGroundCheck : MonoBehaviour
+{
+    [SerializeField] private Transform m_groundCheckTF;
+    [SerializeField] private float m_groundCheckRadius = 0.2f;
+    [SerializeField] private LayerMask m_groundLayer;
+
+    public bool IsGrounded()
+    {
+        Collider[] hits = Physics.OverlapSphere(m_groundCheckTF.position, m_groundCheckRadius, m_groundLayer);
+
+        if (hits.Length > 0)
```

Comment:
Add a gizmos display to the overLapSphere that we can disable and enable easily. Perhaps by having a layer for this gizmos in the gizmos tab. We should be able to easily visualize the ground check sphere.